### PR TITLE
mgr/telemetry: fix 'telemetry {on,off}'

### PR DESCRIPTION
**Pull request by liewegas**
Fixes: https://tracker.ceph.com/issues/38731

Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

**Comment by LenzGr** [comment]
Adding @votdev to double check that the dashboard's manager module configuration tool copes with this change (I assume it does)

**Comment by liewegas** [comment]
retest this please

**Comment by tchaikov** [comment]
retest this please.

File: src/pybind/mgr/telemetry/module.py [hunk 1]
```diff
@@ -272,10 +272,10 @@ def handle_command(self, inbuf, command):
                 r[opt['name']] = getattr(self, opt['name'])
             return 0, json.dumps(r, indent=4), ''
         elif command['prefix'] == 'telemetry on':
-            self.set_config('active', True)
+            self.set_module_option('enabled', True)
```
Comment (votdev) [inline src/pybind/mgr/telemetry/module.py]: What about using localized function here? Thus it can be enabled/disabled for specific managers.

**Comment by votdev** [comment]
@liewegas Could you please address https://github.com/ceph/ceph/pull/26943/files#diff-79a1f16d7dcface908b84ac6ec584cf4R283 in your PR, too?

File: src/pybind/mgr/telemetry/module.py (hunk 1 above)
Comment (liewegas) [reply src/pybind/mgr/telemetry/module.py]: Oops, just saw this.  What's the use case for that?

Comment (votdev) [reply src/pybind/mgr/telemetry/module.py]: No prob, now i think it's a silly idea which does not make sense.

**Pull request merged**